CONTRIBUTING.md

```diff
@@ -188,8 +188,8 @@ can use this syntax to run it exactly as the test harness would:
 $ python tools/test.py --mode=release parallel/test-stream2-transform
 ```
 
-If you want to check the other option, please refer to the help with the option
- `--help`
+If you want to check the other options, please refer to the help with the
+option `--help`
```

Comment:
Nit: "please refer to the help with the option `--help`" => "please refer to the documentation by using the `--help` option"